python/requirements.txt

```diff
@@ -1,7 +1,7 @@
 ecdsa>=0.9
 mnemonic>=0.20
 requests>=2.4.0
-click>=7,<8.1
+click>=7,<8.2
```

Comment:
Maybe we don't want to be strict here, or do we expect that click will break the API again in 8.2?

```suggestion
click>=7
```

Reply:
> do we expect that click will break the API again in 8.2?

given their history, absolutely yes 👀